.gitignore

```diff
@@ -1,8 +1,10 @@
 .DS_Store
+coverage/*
 node_modules
 index.build.js
 es
 umd
 /*.js
+!setupTests.js
```

Comment:
Why are we ignoring this?

Reply:
we are not ignoring this, kindly look it proper